src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,164 @@
+package duke.parser;
+
+import duke.commands.Command;
+import duke.exceptions.*;
+import duke.tasks.*;
```

Comment:
Similar to previous: shouldn't be using wildcard imports. Consider importing only as needed!

Reply:
Agreed, but I think there might be some careless autocorrecting by IntelliJ? On my version of IntelliJ, auto importing all the classes within a package will automatically collapse the import statement to use *?